chore: merge in starbase (#163)
canonical/craft-archives: docs/index.rst

Craft Archives
==============

> Craft Archives is a Python package to manage interaction with software archives on
> behalf of tools using the Craft Parts framework.

Reviewer: ```suggestion
Craft Archives is a Python package that manages interactions with software package
repositories on behalf of tools that use the Craft Parts library.
```
